ext/stringio/stringio.c

```diff
@@ -1428,15 +1428,157 @@ strio_readline(int argc, VALUE *argv, VALUE self)
 }
 
 /*
+ * :markup: markdown
+ *
  * call-seq:
- *   each_line(sep = $/, chomp: false) {|line| ... }   -> self
- *   each_line(limit, chomp: false) {|line| ... }      -> self
- *   each_line(sep, limit, chomp: false) {|line| ... } -> self
- *
- * Calls the block with each remaining line read from the stream;
- * does nothing if already at end-of-file;
- * returns +self+.
- * See {Line IO}[rdoc-ref:IO@Line+IO].
+ *   each(sep = $/, chomp: false) {|line| ... }   -> self
+ *   each(limit, chomp: false) {|line| ... }      -> self
+ *   each(sep, limit, chomp: false) {|line| ... } -> self
```

Comment:
Could you keep using `each_line`?
`each_line` is preferred over `each`.

Reply:
I agree entirely

 Can we switch the alias relationship between `#each` and `#each_line`?  If so, I can do that here (along with changing the call-seq and examples);  that would change this PR to not-[DOC].

Would that be okay?